gnosis/safe-react
# Bump bnc-onboard to v1.35

## What it solves
Add some fixes related to Wallet Connect and L2 networks. Also improves compatibility with EIP-1559 in general
It also includes changes from #2872

## How to test
 - Check that all wallets can be used in Rinkeby and any L2 chain
 - Check that this issue is not happening any more #2092 
 - Check that this issue is not happening any more #2740
